docs/tags.md

```diff
@@ -0,0 +1,30 @@
+# Tags
```

Comment:
The whole content of this file does not belong to this project

Reply:
Yes, that's true, but I find it interesting to mention it nonetheless! I will add it to the runner-manager documentation then.